src/igvfd/searches/configs/file_set.py

```diff
@@ -7,12 +7,45 @@
 def file_set():
     return {
         'facets': {
-            'collections': {
-                'title': 'Collections',
+            'file_set_type': {
+                'title': 'File Set Type',
+            },
+            'assenbly': {
```

Comment:
sp: `assembly`. This is spelled correctly in the facet groups.

Reply:
done